src/test/java/org/takes/facets/auth/social/PsGoogleTest.java

```diff
@@ -113,13 +147,13 @@ public Response act(final Request req) throws IOException {
                         );
                         return new RsJson(
```

Comment:
@dalifreire if we extract the creation of this `RsJson` into a private method, we could get rid of `IMAGE` and `URL`, reducing coupling between tests. This method should expect the `id` as a parameter.